spec/src/main/asciidoc/converters.asciidoc

```diff
@@ -104,3 +104,19 @@ If no built-in nor custom `Converter` exists for a requested Type `T`, an implic
 
 If a `Converter` implements the `java.lang.AutoCloseable` interface  then the `close()` method will be called when the underlying `Config` is being released.
 
+=== Empty value
+
+A config property can be configured with an empty value (the empty String `""`).
+Such config property will be interpreted by the Config API as follows:
+
+* `Config.getValue(key, String.class)` returns `""` (empty String)
+* `Config.getValue(key, String[].class)` returns a 0-sized `String[]` array
```

Comment:
Having an empty string be an empty string is implicitly contradictory with this requirement.  If `,` returns an array of `[ "", "" ]` then the empty string must return an array of `[ "" ]`.  The only logical and consistent solution is to have arrays (and lists) ignore empty strings and have an empty string be identical to "not present".